cli/flox/src/main.rs

```diff
@@ -60,8 +60,8 @@ fn main() -> ExitCode {
             .unwrap_or_default()
     };
 
-    init_logger(Some(verbosity));
     let _sentry_guard = init_sentry();
+    init_logger(Some(verbosity));
```

Comment:
This reordering is necessary to prevent a panic.

Reply:
what was the panic, if you remember?

Reply:
I don't remember exactly, it was related to initializing the Sentry tracing layer without having initialized Sentry. I think we only ran into it once we started sending spans.